presto-docs/src/main/sphinx/connector/mysql.rst

```diff
@@ -66,6 +66,9 @@ Property Name                                      Description
 
 ``case-insensitive-name-matching.cache-ttl``       Duration for which remote dataset and table names will be
                                                    cached. Set to ``0ms`` to disable the cache.                         ``1m``
+
+``mixed-case-support``                             Enable mixed case identifier support for schema and table names      ``false``
```

Comment:
nit: add a period at the end of description and maybe the default value should be on the last line of the description.